osmoutils/accum/errors.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 var (
-	ZeroSharesError = errors.New("shares must be non-zero")
+	ErrZeroShares = errors.New("shares must be non-zero")
```

Comment:
This seems unrelated. It's not a problem, just don't see why it needed to be changed in this pr

Reply:
Lint problem. Basically by bringing osmoutils / osmomath into our repo, it started to get linted following our existing linting rules. Got bunch of linting problems that I needed to fix